### PR TITLE
error on empty arguments

### PR DESCRIPTION
**Pull request by bkamins**
fixing #17 by throwing `ArgumentError`.

### CODE REVIEW

File: src/freqtable.jl [hunk 1]
```diff
@@ -89,6 +91,8 @@ freqtable{T<:Real}(x::AbstractVector...;
 # Internal function needed for now so that n is inferred
 function _freqtable{n}(x::NTuple{n, AbstractCategoricalVector}, skipmissing::Bool = false)
     len = map(length, x)
+    lenn == 0 && throw(ArgumentError("at least one argument must be provided"))
```
Comment (nalimilan) [inline src/freqtable.jl]: Typo.

Comment (nalimilan) [reply src/freqtable.jl]: Also, shouldn't this be `n == 0`?

Comment (bkamins) [reply src/freqtable.jl]: it should, copy-pase 😭. fixed

**Comment by nalimilan** [comment]
Thanks!

**Pull request merged**